.buildkite/scripts/run_models.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+
+apt-get update
+apt-get install libsasl2-dev
+
+python3 -m venv venv
+. venv/bin/activate
+pip install --upgrade pip setuptools
+pip install -r integration_tests/requirements.txt
+mkdir -p ~/.dbt
+cp integration_tests/ci/sample.profiles.yml ~/.dbt/profiles.yml
+
+db=$1
+echo `pwd`
+cd integration_tests
+dbt deps
+dbt seed --target "$db" --full-refresh
+dbt run --target "$db" --full-refresh
```

Comment:
do we wanna include a run where keywords are [disabled](https://github.com/fivetran/dbt_twitter_source#step-4-disabling-keyword-models)?

Reply:
Good callout, just updated!